### PR TITLE
✨ Add GfK Sensic to AMP Analytics Vendors

### PR DESCRIPTION
**Pull request by Spengreb**
✨ Add [GfK Sensic](http://sensic.net/) to amp-analytics vendors

Made with help from @simonArnold

### CODE REVIEW

**Comment by Spengreb** [comment]
@googlebot I signed it!

**Comment by Spengreb** [comment]
@googlebot I signed it!

File: examples/analytics-vendors.amp.html [hunk 1]
```diff
@@ -655,6 +656,19 @@
 </amp-analytics>
 <!-- End Facebook Pixel Analytics example -->
 
+<!-- GfK Sensic tracking -->
+<amp-analytics type="gfksensic" id="gemius">
```
Comment (simonArnold) [inline examples/analytics-vendors.amp.html:660]: @Spengreb This ID of gemius is wrong I think :)

**Comment by Spengreb** [comment]
Hi, is someone able to review this change?

**Comment by Spengreb** [comment]
Hi, I hope someone is able to look at this PR soon. We would like to get this merged as soon as possible. Any feedback is appreciated.

**Pull request closed without merging**